src/components/layout.css

```diff
@@ -5,4 +5,6 @@
 
 @import "../styles/global.css";
 
+@import "../../node_modules/@fortawesome/fontawesome-svg-core/styles.css";
```

Comment:
It'd be nice to have a cleaner import similar to `@tailwind`, but i did not dive too deep into postcss fortawesome